### PR TITLE
Release/1.1.0

### LINKED ISSUE #33: The plugin hasn't been tested with the latest version of WordPress
There is a new WordPress version that the plugin hasn't been tested with. Please test it and then change the "Tested up to" field in the plugin readme.

**Tested up to:** 6.0
**Latest version:** 6.3

This issue will be closed automatically when the versions match.

### LINKED ISSUE #34: Release version 1.1.0
This issue is for tracking changes for the 1.1.0 release.  Target release date: **August 2023**.

### [Release steps](https://github.com/10up/simple-google-news-sitemap/blob/develop/CONTRIBUTING.md#release-instructions)
- [x] Branch: Starting from `develop`, cut a release branch named `release/1.1.0` for your changes.
- [x] Version bump: Bump the version number in `simple-google-news-sitemap.php`, `readme.txt`, `package.json`, `package-lock.json` and any other relevant files if it does not already reflect the version being released.
- [x] Changelog: Add/update the changelog in `CHANGELOG.md` and `readme.txt`.
- [x] Props: update `CREDITS.md` file with any new contributors, confirm maintainers are accurate.
- [x] New files: Check to be sure any new files/paths that are unnecessary in the production version are included in `.distignore`.
- [x] Readme updates: Make any other readme changes as necessary. `README.md` is geared toward GitHub and `readme.txt` contains WordPress.org-specific content.  The two are slightly different.
- [x] Merge: Make a non-fast-forward merge from your release branch to `develop` (or merge the pull request), then do the same for `develop` into `trunk` (`git checkout develop && git pull origin develop && git checkout trunk && git merge --no-ff develop`).  `trunk` contains the stable development version.
- [x] Push: Push your `trunk` branch to GitHub (e.g. `git push origin trunk`).
- [x] Compare `trunk` to `develop` to ensure no additional changes were missed. Visit https://github.com/10up/simple-google-news-sitemap/compare/trunk...develop.
- [x] Test the pre-release ZIP locally by downloading it from the **Build release zip** action artifact and installing it locally.  Ensure this zip has all the files we expect, that it installs and activates correctly and that all basic functionality is working.
- [x] Release: Create a [new release](https://github.com/10up/simple-google-news-sitemap/releases/new), naming the tag and the release with the new version number, and targeting the `trunk` branch.  Paste the changelog from `CHANGELOG.md` into the body of the release and include a link to the closed issues on the [milestone](https://github.com/10up/simple-google-news-sitemap/milestone/5?closed=1).
~- [ ] SVN: Wait for the [GitHub Action](https://github.com/10up/simple-google-news-sitemap/actions/workflows/wordpress-plugin-deploy.yml) to finish deploying to the WordPress.org repository.  If all goes well, users with SVN commit access for that plugin will receive an emailed diff of changes.~
~- [ ] Check WordPress.org: Ensure that the changes are live on https://wordpress.org/plugins/simple-google-news-sitemap/.  This may take a few minutes.~
- [x] Close milestone: Edit the [milestone](https://github.com/10up/simple-google-news-sitemap/milestone/5) with release date (in the `Due date (optional)` field) and link to GitHub release (in the `Description` field), then close the milestone.
- [x] Punt incomplete items: If any open issues or PRs which were milestoned for `1.1.0` do not make it into the release, update their milestone to `1.2.0` or `Future Release`.

### PR DESCRIPTION
**Pull request by dkotter**
### Description of the Change

Version bump and prep for a 1.1.0 release.

Note this also bumps the "tested up to" version to 6.3 as I tested things there and didn't find any problems.

Closes #34

### How to test the Change

Verify all file changes look accurate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
+[![Support Level](https://img.shields.io/badge/support-beta-blueviolet.svg)](#support-level) [![Release Version](https://img.shields.io/github/release/10up/simple-google-news-sitemap.svg)](https://github.com/10up/simple-google-news-sitemap/releases/latest) ![WordPress tested up to version](https://img.shields.io/badge/WordPress-v6.3%20tested-success.svg) [![License](https://img.shields.io/github/license/10up/simple-google-news-sitemap.svg)](https://github.com/10up/simple-google-news-sitemap/blob/develop/LICENSE.md) [![Dependency Review](https://github.com/10up/simple-google-news-sitemap/actions/workflows/dependency-review.yml/badge.svg)](https://github.com/10up/simple-google-news-sitemap/actions/workflows/dependency-review.yml) [![Linting](https://github.com/10up/simple-google-news-sitemap/actions/workflows/lint.yml/badge.svg)](https://github.com/10up/simple-google-news-sitemap/actions/workflows/lint.yml) [![Test](https://github.com/10up/simple-google-news-sitemap/actions/workflows/test.yml/badge.svg)](https://github.com/10up/simple-google-news-sitemap/actions/workflows/test.yml)
```
Comment (jeffpaul) [inline README.md:5]: Maybe I'm missing something, but what changed here?

**Review by jeffpaul — APPROVED** [review]
Looks good, thanks for handling the release!

File: README.md (hunk 1 above)
Comment (dkotter) [reply README.md:5]: WordPress tested up to version link: `https://img.shields.io/badge/WordPress-v6.3%20tested-success.svg`. Went from v6.0 to v6.3 in that URL

Comment (jeffpaul) [reply README.md:5]: Aha, right, that tiny bit that's here because this isn't on DOTORG.  Didn't show in the red/green compare.  Thanks for clarifying!

**Pull request merged**